Makefile.am

```diff
@@ -413,9 +413,11 @@ sssdlib_LTLIBRARIES += \
     libsss_ad.la
 endif
 
+if BUILD_ID_PROVIDER_IDP
```

Comment:
At the very least building of 'sssd-idp' man page should be disabled as well.

Reply:
And also something like `build_idp` is required in sssd.spec.in - to wrap `%{_libdir}/%{name}/libsss_idp.so` and `%{_mandir}/man5/sssd-idp.5*`

Maybe something like
```
%if 0%{?fedora} >= 43
%global build_idp 1
```

Reply:
I made the build of the sssd-idp man page conditional and added `build_idp` to the spec file. So far I made it unconditionally `1` to have a better CI coverage.

Reply:
I didn't even know it's possible to use "--with-id-provider-idp=no".
I always thought it is just "--with..." / "--without...".